ingest/ingest_pipeline.py

```diff
@@ -8,18 +8,7 @@
 PREREQUISITES
 See https://github.com/broadinstitute/scp-ingest-pipeline#prerequisites
 
-DEVELOPER SETUP (see also ../scripts/setup_mongo_dev.sh)
-To run ingest_pipeline on the command line, set the following environment variable(s):
-Set up access to your developer MongoDB instance
-    export MONGODB_USERNAME='single_cell'
-    export DATABASE_NAME='single_cell_portal_development'
-leverage Vault to securely set up the following environment variable(s):
-    MONGODB_PASSWORD
-    DATABASE_HOST
-Bypass MongoDB check for ingest file info in MongoDB and bypass writing results to MongoDB
-    export BYPASS_MONGO_WRITES='yes'
-(optional) To have validation events report to Mixpanel, set:
-    export BARD_HOST_URL="https://terra-bard-dev.appspot.com"
+DEVELOPER SETUP (see README.md/Install and ../scripts/setup_mongo_dev.sh)
```

Comment:
More copy-pasteable into browser address bar:

```suggestion
DEVELOPER SETUP (see README.md#Install and ../scripts/setup_mongo_dev.sh)
```

Reply:
Had not realized that was possible, thanks! `d18014a`